lib/views/recipe_list_screen.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../controllers/recipe_bloc.dart';
+import '../controllers/recipe_event.dart';
+import '../controllers/recipe_state.dart';
+import '../models/recipe.dart';
+import '../constants.dart';
+import 'recipe_detail_screen.dart';
+
+class RecipeListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Recipes'),
+        backgroundColor: kPrimaryColor,
+      ),
+      body: BlocProvider(
+        create: (context) => RecipeBloc()..add(FetchRecipes()),
+        child: RecipeListView(),
+      ),
+    );
+  }
```

Comment:
Nit: All dependency should be allow to inject via constructor.
```suggestion
class RecipeListScreen extends StatelessWidget {
  const RecipeListScreen({
    super.key,
    @visibleForTesting this.recipeBloc,
  });
  final RecipeBloc? recipeBloc;

  @override
  Widget build(BuildContext context) {
    const receiptListView = RecipeListView();

    return Scaffold(
      appBar: AppBar(
        title: const Text('Recipes'),
        backgroundColor: kPrimaryColor,
      ),
      body: recipeBloc == null
          ? BlocProvider(
              create: (context) => RecipeBloc()..add(FetchRecipes()),
              child: receiptListView,
            )
          : BlocProvider.value(
              value: recipeBloc!,
              child: receiptListView,
            ),
    );
  }
}
```

It's best to write a helper function to handle BlocProvider for this case.